packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1017,7 +1015,7 @@ export class StripeHelper extends StripeHelperBase {
         try {
           const { currency, discount, total, total_discount_amounts } =
             await this.previewInvoice({
-              automaticTax,
+              automaticTax: true,
```

Comment:
The automatic tax feature flagging behavior needs to be removed within previewInvoice as well.